src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/bulk.html.twig

```diff
@@ -21,18 +21,19 @@
  * @author    PrestaShop SA and Contributors <contact@prestashop.com>
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
- *#}
+ * #}
 {% set legacyBaseLayout = 'light_display_layout.tpl' %}
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 
-{% form_theme bulkCombinationForm '@PrestaShop/Admin/Sell/Catalog/Product/FormTheme/bulk_combination.html.twig' %}
+{% form_theme bulkCombinationForm '@PrestaShop/Admin/Sell/Catalog/Product/FormTheme/combination.html.twig' %}
 
 {% block stylesheets %}
-  <link rel="stylesheet" href="{{ asset('themes/new-theme/public/product' ~ rtl_suffix ~ '.css') }}" type="text/css" media="all">
+  <link rel="stylesheet" href="{{ asset('themes/new-theme/public/product' ~ rtl_suffix ~ '.css') }}" type="text/css"
+        media="all">
 {% endblock %}
 
 {% block content %}
-  {{ form_start(bulkCombinationForm) }}
+  {{ form_start(bulkCombinationForm, {'attr': {'class': 'combination-page ' ~ bulkCombinationForm.vars.attr.class|default('')}}) }}
```

Comment:
It might be cleaner to set this class in the form type in the end But for now it should be good, because I need your modification that integrate productId in the URL ^^